openedx/features/wikimedia_features/meta_translations/apps.py

```diff
@@ -17,3 +17,6 @@ class MetaTranslationsConfig(AppConfig):
             }
         }
     }
+
+    def ready(self):
+        from . import signals  # pylint: disable=unused-import
```

Comment:
Add a new line at the end for git convention

Reply:
Oky.